clients/githubrepo/branches.go

```diff
@@ -340,6 +340,13 @@ func copyAdminSettings(src *branchProtectionRule, dst *clients.BranchProtectionR
 			copyBoolPtr(&upToDateBeforeMerge, &dst.CheckRules.UpToDateBeforeMerge)
 		}
 	}
+	if readBoolPtr(src.DismissesStaleReviews) {
```

Comment:
more info: when running with an admin token, without PRs, what's the value of `src.DismissesStaleReviews`?

i'm curious if this needs to change to support cases where `src.DismissesStaleReviews` is non-nil and false.